release-notes/aws/v9.1.1.md

```diff
@@ -0,0 +1,16 @@
+## :zap: Giant Swarm Release 9.1.1 for AWS is now active for you! :zap:
+
```

Comment:
```suggestion
This release upgrades the NGINX IC app to upstream v0.28.0. It contains several non-breaking changes that fix a few issues. We also added metrics Service for prometheus-operator support. This enables scraping of metrics and better monitoring on the app.
```

Reply:
WDYT about this? 
Also, what can the customer do here - any CTA we want to add? Like, should they upgrade, talk to SE... or just fyi?

Reply:
Applied suggestion as is, likely IC abbreviation should be replaced with Ingress Controller.
Even more likely we need to add a note related to https://github.com/giantswarm/giantswarm/issues/8851#issuecomment-583723774 - can you please come up with a suggestion? Or would you like me to propose initial content for the note? Same needs to be communicated for https://github.com/giantswarm/releases/pull/89